qgroundcontrol.pro

```diff
@@ -1403,3 +1403,39 @@ LinuxBuild {
 
     INSTALLS += target share_qgroundcontrol share_icons share_metainfo share_applications
 }
+
+
+#-------------------------------------------------------------------------------------
+#For 3D Viewer
+
+QT += \
+    quick3d \
+    xml
+
+CONFIG += qmltypes
+QML_IMPORT_NAME = Viewer3DQmlType
+QML_IMPORT_MAJOR_VERSION = 1
+
+
+SOURCES += \
```

Comment:
Same thing with this stuff. Since this will just become part of all regular QGC builds. This should just be where all the other source is added. We only tend to split things out like this when that feature is optional.

Reply:
Sure. I will move sources and the modules to the right places.